src/app/portlets/dot-edit-page/main/dot-edit-page-nav/dot-edit-page-nav.component.html

```diff
@@ -1,8 +1,19 @@
 <ul class="edit-page-nav">
     <li *ngFor="let item of model | async">
-        <a class="edit-page-nav__item" [class.edit-page-nav__item--disabled]="item.disabled" routerLinkActive="edit-page-nav__item--active" [routerLink]="item.link" [queryParams]="route.queryParams | async">
-            <i class="{{ item.icon }}"></i>
-            <span class="edit-page-nav__item-text">{{ item.label }}</span>
-        </a>
+        <div *ngIf="!item.needslicense;then basic else enterprise"></div>
+        <ng-template #basic>
+            <a class="edit-page-nav__item" [class.edit-page-nav__item--disabled]="item.disabled" routerLinkActive="{{ !item.disabled ? 'edit-page-nav__item--active' : '' }}"
+                [routerLink]="!item.disabled ? item.link : null" [queryParams]="route.queryParams | async">
+                <i class="{{ item.icon }}"></i>
+                <span class="edit-page-nav__item-text">{{ item.label }}</span>
+            </a>
+        </ng-template>
+        <ng-template #enterprise>
```

Comment:
cann't We  create a directive for this?, somethink like: 

<ng-template license='enteprise'>
....
</ng-template>

and the  if the enterprise or higher the this block  is visible.

Reply:
This is official angular if else in template, coming from the doc.